### DISCUSSION TITLE
wildfly supported version

### DISCUSSION THREAD

**Discussion by hanuamantha** [Keyfactor/ejbca-ce#355] [Q&A]
What is the latest supported WildFly version for EJBCA 8? Is WildFly 29 or 28 supported?

**Comment by primetomas** [comment]
See here: https://doc.primekey.com/ejbca/ejbca-installation/application-servers

WildFly > 26 has breaking changes, Jakarta EE upgrade that is not backwards compatible (as it has always been in the past 20 years). Therefore that upgrade requires full environment re-build, for upgrading EJBCA when that update is made. Therefore this change is held off for a while. There need to be stable and common JBoss EAP etc versions as well. There will be notifications coming when this major breaking upgrade is coming out. Not in short time.
